src/App.jsx

```diff
@@ -1,6 +1,19 @@
-import { ChakraProvider, Button, Image, Card, CardBody, Text } from '@chakra-ui/react';
+import {
+  ChakraProvider,
+  Button,
+  Image,
+  Card,
+  CardBody,
+  Text,
+  useDisclosure,
+  Modal,
+  ModalContent,
+  ModalBody,
+  ModalFooter,
+} from '@chakra-ui/react';
 import React, { useState, useEffect } from 'react';
 import { BrowserRouter as Router, Routes, Route, Outlet } from 'react-router-dom';
+import { PDFViewer } from '@react-pdf/renderer';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module '@react-pdf/renderer'.